tests/test_qhub.py

```diff
@@ -144,7 +144,7 @@ def get_side_effect(path, **kwargs):
     assert not qhub.check_ref_clock(sleep_time=0.001)
     # Locking error and actual_clock != clock => reset clock to internal
     source = 1
-    mock_connection.return_value.syncSetString.assert_not_called()
+    # mock_connection.return_value.syncSetString.assert_not_called()
```

Comment:
Is this line supposed to be removed or was the comment forgotten?

Reply:
I still see it in the latest changes though.